workers/unity/Packages/com.improbable.gdk.core/MutableView/MutableView.cs

```diff
@@ -19,7 +20,7 @@ public class MutableView : IDisposable
 
         private Action<Entity, long> addAllCommandRequestSenders;
 
-        private readonly EntityManager entityManager;
+        public readonly EntityManager EntityManager;
```

Comment:
Note: Temporary until we figure out a worker abstraction

Reply:
Can we just get rid of almost all of these anyway?

Reply:
Yes - out of scope of this PR though

Reply:
Am planning to do a cleanup pass